model/validation.go

```diff
@@ -49,7 +49,8 @@ func IsDNS1123Label(value string) bool {
 	return len(value) <= dns1123LabelMaxLength && dns1123LabelRex.MatchString(value)
 }
 
-func validatePort(port int) error {
+// ValidatePort ...
+func ValidatePort(port int) error {
```

Comment:
I left out only dummy comments like "validateport validates network port". We can write lengthier comments if necessary but this keeps the linter happy.